posthog/permissions.py

```diff
@@ -0,0 +1,58 @@
+from typing import Optional
+
+from django.db.models import Model
+from rest_framework.permissions import SAFE_METHODS, BasePermission
+from rest_framework.request import Request
+
+from posthog.models import Organization, OrganizationMembership, organization
+
+CREATE_METHODS = ["POST", "PUT"]
+
+
+def extract_organization(object: Model) -> Organization:
```

Comment:
This seems like it would go in utils maybe?

Reply:
It's very very permissions specific though, I feel like it'd just bloat utils when permissions is where it makes the most sense.